doc/contributing/distribution.md

```diff
@@ -25,3 +25,8 @@ only as a semver-major change, unless the related feature or project is
 documented as experimental. In addition, Node.js includes external projects as
 internal dependencies. These may be replaced or removed at any time, provided
 that doing so is not a breaking change.
+
+## Executables
+
+The Node.js distribution includes only executables whose names refer to software
+that is vendored within the Node.js distribution.
```

Comment:
I don't think this sentence is understandable as written.

I created a poll to see if others agree: https://twitter.com/styfle/status/1767924753405190631

Reply:
> The Node.js distribution includes only executables whose names refer to software that is vendored within the Node.js distribution. For example, Node.js includes an executable `npm` to run the npm software included within the Node.js distribution; Node.js does not include a `foo` executable which would download software named Foo.

Reply:
I added a second sentence with an example, @styfle please let me know if you still think it’s confusing.